pkg/api/dtos/index.go

```diff
@@ -25,6 +25,8 @@ type IndexViewData struct {
 	AppleTouchIcon          template.URL
 	AppTitle                string
 	Sentry                  *setting.Sentry
+	// Nonce is a cryptographic identifier for use with Content Security Policy.
+	Nonce string
```

Comment:
Weird formatting here, no?

Reply:
@wbrowne How is it weird? The file gets auto-formatted by goimports.

Reply:
Ah, you're referring to the field alignment. Probably unaligned because of the comment above.